page-components/expandables.md

```diff
@@ -28,25 +28,28 @@ Expandables are used for displaying non-essential information on a page.
       <p>Expandables in the Capital Framework</p>
     </li>
   </ul>
-</div> 
+</div>
 
 <h2 id="use">Use<span class="cf-code-link"><a href="https://cfpb.github.io/cf-expandables/docs/">View code <span class="cf-icon cf-icon-external-link"></span></a></span></h2>
```

Comment:
Change link to https://cfpb.github.io/capital-framework/components/cf-expandables/